test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -416,6 +416,52 @@ public void execute_find_onlyMatchesFullWordsInNames() throws Exception {
                                 expectedList);
     }
 
+    @Test
+    public void execute_sort_lexicographical() throws Exception {
+        TestDataHelper helper = new TestDataHelper();
+        Person alice = helper.generatePersonWithName("Alice");
+        Person bob = helper.generatePersonWithName("bob");
+        Person charlie = helper.generatePersonWithName("charlie");
+        Person douglas = helper.generatePersonWithName("douglas");
+
+        List<Person> personsSorted = helper.generatePersonList(alice, bob, charlie, douglas);
+        AddressBook expectedAB = helper.generateAddressBook(personsSorted);
+
+        List<Person> personsUnsorted = helper.generatePersonList(douglas, charlie, bob, alice);
+        helper.addToAddressBook(addressBook, personsUnsorted);
+
```

Comment:
Very minor nitpick on the variable name.
Could've been sortedPersons and notSortedPersons just for easier reading
(Unsorted makes it sound like it has been processed)